src/context/GeoStylerContext/GeoStylerContext.tsx

```diff
@@ -84,7 +82,26 @@ export const useGeoStylerLocale = <T extends keyof GeoStylerLocale>(key: T): Geo
   return structuredClone(ctx.locale[key]);
 };
 
-export const useGeoStylerUnsupportedProperties = (): any => {
+export const useGeoStylerUnsupportedProperties = <T extends Symbolizer>(symbolizer: T) => {
```

Comment:
Do we have to extent this to `Style`? E.g. the props of a Rule might also differ between two parsers

Reply:
Correct. Currently this works for the Symbolizer Editors.